test/unit-test/ota_utest.c

```diff
@@ -434,6 +475,12 @@ static void otaGoToStateWithTimeout( OtaState_t state,
         otaInitDefault();
     }
 
+    /* Default to the MQTT job doc. */
+    if( pOtaJobDoc == NULL )
+    {
+        pOtaJobDoc = JOB_DOC_A;
```

Comment:
does this need to be a global?

Reply:
`pOtaJobDoc`? It can be a parameter to `otaReceiveJobDocument`, but I did this way to control if I'm testing MQTT or HTTP, also avoid changing the signature of `otaGoToState`.